build/Common.test.props

```diff
@@ -15,4 +20,28 @@
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)/stylecop.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests'))">
+    <IsTestProject>true</IsTestProject>
+    <CollectCoverage Condition="$(CollectCoverage) == ''">true</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <CollectCoverage Condition="$(CollectCoverage) == '' OR !$(TargetFramework.StartsWith('netcoreapp'))">false</CollectCoverage>
+  </PropertyGroup>
+
+  <PropertyGroup Label="CollectCodeCoverageResults" Condition="$(CollectCoverage)">
```

Comment:
where would we see coverage numbers? Please add a note in `CONTRIBUTING.md`

Reply:
Sure, i'll add later for you. If you run `dotnet test` the output is generated into the console for you.